<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>394</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Times New Roman</family>
    <pointsize>15</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>331</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Social media information about a person</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="app_progress_bar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>671</width>
      <height>31</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="submit_button">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>300</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Submit</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>44</y>
      <width>671</width>
      <height>252</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>First name</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="3">
      <widget class="QLineEdit" name="first_name_field"/>
     </item>
     <item row="0" column="4" colspan="2">
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Education</string>
       </property>
      </widget>
     </item>
     <item row="0" column="6" colspan="2">
      <widget class="QLineEdit" name="school_field"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_6">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Last name</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="3">
      <widget class="QLineEdit" name="last_name_field"/>
     </item>
     <item row="1" column="4">
      <widget class="QLabel" name="label_8">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Twitter</string>
       </property>
      </widget>
     </item>
     <item row="1" column="5" colspan="3">
      <widget class="QLineEdit" name="twitter_field"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_9">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Location</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="3">
      <widget class="QLineEdit" name="location_field"/>
     </item>
     <item row="2" column="4" colspan="2">
      <widget class="QLabel" name="label_7">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Instagram</string>
       </property>
      </widget>
     </item>
     <item row="2" column="6" colspan="2">
      <widget class="QLineEdit" name="instagram_field"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Company</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="3">
      <widget class="QLineEdit" name="company_field"/>
     </item>
     <item row="3" column="4" colspan="3">
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Additional info</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Job title</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1" colspan="3">
      <widget class="QLineEdit" name="job_title_field"/>
     </item>
     <item row="5" column="0" colspan="2">
      <widget class="QLineEdit" name="extra_field_name"/>
     </item>
     <item row="5" column="2" colspan="2">
      <widget class="QLineEdit" name="extra_field_input"/>
     </item>
     <item row="6" column="0" colspan="3">
      <widget class="QLabel" name="label_10">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>15</pointsize>
         <italic>true</italic>
        </font>
       </property>
       <property name="text">
        <string>PDF output directory</string>
       </property>
      </widget>
     </item>
     <item row="6" column="8">
      <widget class="QPushButton" name="select_directory">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="6" column="3" colspan="5">
      <widget class="QLineEdit" name="directory_text"/>
     </item>
     <item row="3" column="7" rowspan="2">
      <widget class="QTextEdit" name="additional_info_field"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>690</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCredits">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionCredits"/>
   </widget>
   <addaction name="menuCredits"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionCredits">
   <property name="text">
    <string>Credits</string>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>15</pointsize>
    </font>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
